src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final String SUPPLY = "supply,";
+    static final String BUY = "buy,";
+    static final String RESULT = "result,";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromMyFile = new File(fromFileName);
+        File toMyFile = new File(toFileName);
+        try {
+            FileReader fileReader = new FileReader(fromMyFile);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            String report = createReport(bufferedReader);
+
+            FileWriter fileWriter = new FileWriter(toMyFile);
+            BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
+            createFile(bufferedWriter, report);
```

Comment:
```suggestion
            writeToFile(report);
```
lets create fileWriter inside writeToFile method